clicommand/secret_get.go

```diff
@@ -29,9 +29,25 @@ type SecretGetConfig struct {
 }
 
 var SecretGetCommand = cli.Command{
-	Name:        "get",
-	Usage:       "Get a secret by its key",
-	Description: "Get a secret by key from Buildkite Pipelines Secrets and print it to stdout.",
+	Name:  "get",
+	Usage: "Get a secret by its key",
+	Description: `Usage:
+
+    buildkite-agent secret get [key] [options...]
+
+Description:
+
+Gets a secret from Buildkite secrets and prints it to stdout. The ′key′
+specified in this command is the key's name defined for the secret in its
+cluster. The key's name is case insensitive in this command, and the
+key's value is automatically redacted in the build logs.
+
+Examples:
+
+The following examples reference the same Buildkite secret ′key′:
```

Comment:
Would the fact that this line ends in a colon (`:`) cause this line to be rendered as a heading in the docs?

For example, see **Usage**, **Description** and **Example** at the [top of this page of the docs](https://buildkite.com/docs/agent/v3/cli-step#updating-a-step). I suspect not (based on the `To avoid orphaned status updates, in your Pipeline Settings > GitHub:` line on this page) but just wanted to double-check.

Reply:
nope, you're good. i checked this by running the [`scripts/update-agent-help.sh`](https://github.com/buildkite/docs/blob/main/scripts/update-agent-help.sh) script in the docs repo, which produced this output for the `secret get` command:

---
### Usage

`buildkite-agent secret get [key] [options...]`

### Description

Gets a secret from Buildkite secrets and prints it to stdout. The `key`
specified in this command is the key&#39;s name defined for the secret in its
cluster. The key&#39;s name is case insensitive in this command, and the
key&#39;s value is automatically redacted in the build logs.

### Examples

The following examples reference the same Buildkite secret `key`:

```shell
$ buildkite-agent secret get deploy_key
$ buildkite-agent secret get DEPLOY_KEY
```
---

ironically, in typing out my reply, initially the first line was a header. i think so long as there's a newline after the thing that ends with `:`, it won't render it as a header?

Reply:
Thanks for checking this :-)